modules/end-user-guide/pages/url-parameter-for-workspace-image.adoc

```diff
@@ -0,0 +1,14 @@
+:_content-type: CONCEPT
+:description: URL parameter for the workspace image
+:keywords: parameter, URL, workspace, image
+:navtitle: URL parameter for the workspace image
+:page-aliases: url-parameter-for-workspace-image.adoc
```

Comment:
Try this, and the reason is we don't need this yet because the file name hasn't changed yet:
```suggestion
//:page-aliases: url-parameter-for-workspace-image.adoc
```